sunpy/instr/aia/response.py

```diff
@@ -0,0 +1,345 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+__author__ = "Tessa D. Wilkinson"
+
+"""
+AIA response functions by integrating ChiantiPy
+
+    - analyze AIA instrument response that tells efficiency of channel using instrument properties
+    - then output a wavelength response
+    - use spectral model (synthetic spectra) - to get ion emissivities as a function of temp/density
+    - obtain temperature response based on chianti spectral contribution functions (emissivity)
+
+AIA ccd's do not provide spectroscopic information, so there is no way to directly apply a wavelength-dependent calibration to the data.
+
+other important variables:
+effective area
+ion emissivity
+electron density
+temperature array
+
+utilize sunpy/instr/tests/test_aia.py
+
+
+    for units use:
+    @u.quantity_input(wave=u.m)
+
+"""
+
+import numpy as np
+import pandas as pd
+from astropy.table import Table
+import astropy.units as u
+
+import os
+
+import sunpy
+import sunpy.data.test as test
+import aia_read_genx2table
```

Comment:
I get an `ImportError` here. I think this needs to be:
```Python
import sunpy.instr.aia.aia_read_genx2table
```
Alternatively, you could just pull out the function that you need,
```Python
from .aia_read_genx2table import aia_instr_properties_to_table
```
and then change the function call appropriately.